Makefile

```
@@ -1,4 +1,8 @@
-NAME    := argobots
-SRC_EXT := gz
+NAME           := argobots
+SRC_EXT        := gz
+PKG_GIT_COMMIT := 411e5b344642ebc82190fd8b125db512e5b449d1
+GITHUB_PROJECT := pmodels/$(NAME)
+# This list of files that are in the upstream git repo but are not included in upstream's releases
+# PATCH_EXCLUDE_FILES :=
```

If I understand correctly, https://github.com/pmodels/argobots/commit/411e5b344642ebc82190fd8b125db512e5b449d1 is only [1 patch ahead](https://github.com/pmodels/argobots/compare/v1.2..411e5b344642ebc82190fd8b125db512e5b449d1) of https://github.com/pmodels/argobots/releases/tag/v1.2 correct?  If so, the more accepted way of achieving this goal is to back all of this change out and to add the one single commit desired as a `Patch:` in `argbots.spec`.

```suggestion
NAME    := argobots
SRC_EXT := gz
```

---

Yes, this patch is only 1 patch ahead of the release 1.2 
I have used the same approach as for the ior-pkg git repo.
However, I have no issue to change it to what you propose if it is the best practice for such case.

- [ ] update patching method

---

Fixed with commit 69e02db

---

> I have used the same approach as for the ior-pkg git repo.

The significant difference there is the [delta between the 3.3.0 tag and the desired commit (d3574d536643475269d37211e283b49ebd6732d7)](https://github.com/hpc/ior/compare/3.3.0...d3574d536643475269d37211e283b49ebd6732d7) is 294 commits, so using the preferred method of 1 `Patch:` per commit, we'd have 294 `Patch:` entries which gets a bit unruly.

> However, I have no issue to change it to what you propose if it is the best practice for such case.

For a small number of patches, it is much more clear what is being patched when they are kept individually but only to an extent that it's a reasonable (i.e. small) number of patches to inspect individually.

Moreover, patching such a huge delta is really only really acceptable for the ior repository because it's a package for test software which we do not distribute.  Argobots on the other hand is a core piece of the DAOS software stack and as such we should be using official upstream releases as closely as possible.